tests/controller/datacontroller.js

```diff
@@ -339,7 +339,10 @@ describe( 'DataController', () => {
 	describe( 'stringify', () => {
 		beforeEach( () => {
 			modelDocument.schema.registerItem( 'paragraph', '$block' );
-			modelDocument.schema.registerItem( 'div', '$block' );
+			modelDocument.schema.registerItem( 'div' );
+
+			modelDocument.schema.allow( { name: '$block', inside: 'div' } );
+			modelDocument.schema.allow( { name: 'div', inside: '$root' } );
```

Comment:
IIRC, that would mean that it's both a block in which text is allowed and it would need to allow  blocks inside it. So, a schema like this is more realistic. Div is a container which can contain blocks.